src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -888,6 +887,33 @@ public static function get_resolved_theme_uris( $theme_json ) {
 			$resolved_theme_uris[] = $resolved_theme_uri;
 		}
 
+		// Block styles.
+		if ( ! empty( $theme_json_data['styles']['blocks'] ) ) {
+			foreach ( $theme_json_data['styles']['blocks'] as $block_name => $block_styles ) {
+				if ( ! isset( $block_styles['background']['backgroundImage']['url'] ) ) {
+					continue;
+				}
+				$background_image_url = $block_styles['background']['backgroundImage']['url'] ?? null;
+				if (
+					isset( $background_image_url ) &&
+					is_string( $background_image_url ) &&
+					// Skip if the src doesn't start with the placeholder, as there's nothing to replace.
+					str_starts_with( $background_image_url, $placeholder ) ) {
```

Comment:
```suggestion
					str_starts_with( $background_image_url, $placeholder )
				) {
```

Nit: I might have missed it iin the [WP PHP code style guidelines](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#multiline-function-calls) but I think the convention is having each condition entirely on its own line like the multiline function calls.

For me at least, it helps when scanning blocks of code.

Reply:
Easy to fix - phpcs complains locally, but I can see if pushing it breaks anything 👍🏻

Reply:
I tested making the changes locally and my PHPCS and PHPCBF weren't complaining or autoformatting it 🤔 

As I noted, these are nits, so don't let them block moving this forward.

Reply:
It's a me problem I think 🤣